CMakeLists.txt

```
@@ -83,6 +84,10 @@ if(ENABLE_SYSTEM_GLM)
     find_package(GLM REQUIRED)
 endif()
 
+if(ENABLE_SYSTEM_PROJECTM_EVAL)
+    find_package(projectM-eval REQUIRED)
```

Remove the REQUIRED keyword and use correct casing for the config package dir:

```suggestion
    find_package(projectM-Eval)
```